clippy_lints/src/booleans.rs

```diff
@@ -416,7 +418,12 @@ impl<'a, 'tcx> NonminimalBoolVisitor<'a, 'tcx> {
                     NONMINIMAL_BOOL,
                     e.span,
                     "this boolean expression can be simplified",
-                    |db| { db.span_suggestions(e.span, "try", suggestions); },
+                    |db| { db.span_suggestions_with_applicability(
+                            e.span,
```

Comment:
inconsistent formatting (for example)

Reply:
Shall it be further right?

Reply:
`rustfmt` converts this to
```rust
|db| {
    db.span_suggestions_with_applicability(
        e.span,
        "try",
        suggestions,
        Applicability::Unspecified,
    );
},
```

Reply:
I applied `cargo fmt` and it indeed found two span_suggestion things, but not this one...

Is rustfmt and `cargo fmt` the same engine? Or maybe mine is too outdated? (`rustfmt 0.99.4-nightly (1c40881 2018-08-27)`)

Reply:
Yeah they're the same engine. I always use `rustfmt` out of vim and never use `cargo fmt`. But it seems that the `rustfmt.toml` file prevents this things from getting formatted. You could temporary delete this file, then run `cargo fmt` and add the relevant parts with `git add -p`.